base/render_readme_test.go

```diff
@@ -0,0 +1,38 @@
+package base
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/qri-io/qfs"
+)
+
+func TestRenderReadme(t *testing.T) {
```

Comment:
gonna need at least one test showing what happens when you try to add a `<script>` tag to a readme before this can merge. must test contents of results.